app/assets/stylesheets/application.scss

```diff
@@ -25,7 +25,6 @@ $govuk-image-url-function: 'image-url';
 @import 'components/og-preview';
 @import 'components/hiring-staff';
 @import 'components/select';
-@import 'components/tabs';
```

Comment:
Hmm, was this file removed as part of this commit?  I can't see it removed, but I'm pretty sure it can be now.

Reply:
Will remove it and rebase to squash that into this commit

Reply:
addressed in https://github.com/DFE-Digital/teacher-vacancy-service/pull/1057/commits/3e25285328fb63766611b02dc2c48feeae86f178